bindings/go/src/fdb/transaction.go

```diff
@@ -372,6 +372,16 @@ func (t Transaction) GetVersionstamp() FutureKey {
 	return &futureKey{future: newFuture(C.fdb_transaction_get_versionstamp(t.ptr))}
 }
 
+func (t *transaction) getApproximateSize() FutureInt64 {
+	return &futureInt64{
```

Comment:
`futureInt64` is calling `fdb_future_get_version`. I suppose we should have two different types in Go for this, though thinking about it a little more now maybe the best thing is to replace `fdb_future_get_version` with `fdb_future_get_int64` in the C API and not maintain 2 of these?

Reply:
I changed Go's FutureInt64 to use `fdb_future_get_int64`. For C API, I keep both `fdb_future_get_version` and `fdb_future_get_int64`, because the former is needed to be compatible with old code.

Reply:
We have a deprecation mechanism that could be used to remove it, and all the bindings could be updated to only use the int64 variant. I won't argue for or against going that route though.